skip-tests.json

```diff
@@ -23,10 +23,8 @@
             },
             "fetchCurrencies": {
                 "currencyIdAndCode": "https://app.travis-ci.com/github/ccxt/ccxt/builds/269432480#L3364",
-                "withdraw": "not provided",
-                "deposit": "not provided",
-                "type": "todo",
-                "activeMajorCurrencies": "todo"
+                "activeMajorCurrencies": "todo",
+                "allowNull": ["withdraw", "deposit", "type"]
```

Comment:
adding with separate keys affects with larger scale than needed (eg those keys are totally skipped even from structural checs). the `undefined` value tolerance is an isolated (smaller scope) case and it needs to have an additional separate key